runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -328,13 +347,21 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
+		if (config.screenshotCcKick() && chatMessage.startsWith("Your request to kick"))
+		{
+			final String nameText = kickPlayerName == null ? "" : kickPlayerName + " ";
```

Comment:
A space is added after the given filename in `ImageCapture`. Don't add one here.